### DISCUSSION TITLE
Stopping pnpm with command+c stops node process with error message

### DISCUSSION THREAD

**Discussion by moritzlaube** [pnpm/pnpm#7025] [Q&A]
I am new to pnpm and I'm sure this is me, but I'd be grateful if someone could point me in the right direction: 
When I am stopping a Node process (like working with sveltekit in dev mode) with command+c like i would do with npm, I am getting the following error: 
`ELIFECYCLE  Command failed.` 

And when I am in a monorepo project using pnpm to orchestrate the startup of frontend and backend, I am getting the following error on stopping the process: 
` ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @pitch-pal/app@0.0.1 dev: `vite dev`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1.`
 
 Is that a normal error when exiting the process with command+c? Am I doing something wrong? Thanks in advance!

**Comment by chshouyu** [comment]
same issue. very annoying

**Comment by chshouyu** [comment]
Finally, I use npm to start dev server and use pnpm to install dependencies. If you use pnpm workspace, you can use it with npm workspaces, the root `package.json` will be:

```json
{
  "scripts": {
    "dev:backend": "npm run dev -w backend",
    "dev:frontend": "npm run dev -w frontend",
    "dev:all": "concurrently -k -c \"green,cyan\" -n \"backend,frontend\" \"npm run dev:backend\" \"npm run dev:frontend\"",
  },
  "devDependencies": {
    "concurrently": "^8.2.1"
  },
  "workspaces": [
    "apps/*"
  ]
}
```

run `npm run dev:all` in terminal, or run `npm run dev:frontend`, `npm run dev:backend` in different terminal separately.

Also, if you use `vite` alone, you can press `q` in terminal to stop vite dev server peacefully.

**But I still hope that pnpm can solve this problem as soon as possible.**
